src/rgw/rgw_op.cc

```diff
@@ -6981,8 +6981,13 @@ void RGWDeleteMultiObj::handle_individual_object(const rgw_obj_key& o, optional_
   del_op->params.marker_version_id = version_id;
 
   op_ret = del_op->delete_obj(this, y);
-  if (op_ret == -ENOENT) {
-    op_ret = 0;
+  if (op_ret >= 0) {
+    ldpp_dout(this, 1) << "DELETED: bucket=" << bucket->get_name() << ", obj=" << obj->get_oid() << dendl;
```

Comment:
the level 1 is correct for the error case, but not the success case--we want to be consistent with the rest of the file (and level 1 should not be logging informative messages;  if you see cases where we're not following that guidance, please change them)

Reply:
Agreed，I change it to 5.

Reply:
Deleting object maybe different from other operations. Without log, We can't tell users definitively whether data has been deleted by somebody's  request  or due to the system's error. And this is related to service's SLA,  Maybe level 1 is better (though it goes againt rules...), or we can define new level for such operations? @mattbenjamin